Makefile

```diff
@@ -1,7 +1,7 @@
 # Makefile 
 # Run `make` to clean taxons and content.
-# Run `make init` to install required packages with pip.
-# Run `make test` to run tests on pipeline_functions.
+# Run `make pip_install` to install required packages with pip.
```

Comment:
We'll want to change the [README](https://github.com/alphagov/govuk-taxonomy-supervised-learning#setting-environment-variables) description reference as well: 

> ## Preparing your python environment
> The Makefile assumes that the python3 command is pointing to the correct distribution of python, which was 3.4.6 in development. To install the correct package dependencies run **make init** from the project root.

Also, if you're editing the file, perhaps we can fix the heading markdown to render correctly?

Reply:
I'll update the README. I think the headers issue might be fixed with https://github.com/alphagov/govuk-taxonomy-supervised-learning/pull/47 ?

Reply:
I've now updated the README in b3d325b6c36cfa6e7dff5561c7c43e1a737d2ab6